database/migrations/1733096077721_create_contributor_social_links_table.ts

```diff
@@ -0,0 +1,52 @@
+import { BaseSchema } from '@adonisjs/lucid/schema'
+
+export default class extends BaseSchema {
+  protected tableName = 'contributor_social_links'
+  protected linkTypes = [
+    'default',
+    'facebook',
+    'instagram',
+    'linkedin',
+    'mailto:',
+    'youtu',
+    'github',
+    'topwr:buildings',
+    'tel',
+    'https://x.com',
+    'tiktok',
+    'discord',
+    'twitch',
+  ]
+
+  async up() {
+    this.schema.createTable(this.tableName, (table) => {
+      table.increments('id')
+      table
+        .integer('contributor_id')
```

Comment:
it's better to use .foreign imo
https://lucid.adonisjs.com/docs/table-builder#foreign

Reply:
aight, will do

Reply:
i don't think you need to split it into `.integer` and `.foreign` parts
just `.foreign` seems to be doing just fine